docs/templating/render_content/render_content_in_php.md

```diff
@@ -0,0 +1,46 @@
+---
+description: Render Content item from PHP and get the resulting HTML as a string.
+---
+
+# Render content in PHP
+
+While in PHP, you may need to render the view of a Content item (for further treatment like PDF conversion, because you're not in an HTML context, etc.).
+
+!!! caution
+
+    Avoid using this in a controller as much as possible.
```

Comment:
```suggestion
    Avoid using rendering in a controller as much as possible.
```
"this" is not descriptive enough

Reply:
You're right.

In a way, You can render but not by yourself… Let's be more descriptive:

```suggestion
    Avoid using PHP rendering in a controller as much as possible.
```